pypi_formula_mappings.json

```diff
@@ -7,9 +7,10 @@
   },
   "ansible": {
     "extra_packages": [
-      "apache-libcloud", "boto3", "dnspython", "docker", "junos-eznc", "jxmlease", "kerberos", "ntc-templates",
-      "openshift", "passlib", "pexpect", "proxmoxer", "pysphere3", "python-consul", "python-neutronclient",
-      "pywinrm", "requests-credssp", "shade", "zabbix-api"
+      "apache-libcloud", "boto3", "cffi", "dnspython", "docker",
+      "junos-eznc", "jxmlease", "kerberos", "ntc-templates", "openshift",
+      "passlib", "pexpect", "proxmoxer", "pycparser", "pysphere3", "python-consul",
+      "python-neutronclient", "pywinrm", "requests-credssp", "shade", "zabbix-api"
     ],
     "exclude_packages": ["six", "PyYAML"]
```

Comment:
Shouldn't `cffi` and `pycparser` be in `"excluse_packages"` instead?

Reply:
they are in the list.

Reply:
oh, sorry, you are right

Reply:
good catch!